src/services/print.js

```diff
@@ -656,8 +656,11 @@ ngeo.Print.prototype.encodeTextStyle_ = function(symbolizers, textStyle) {
       symbolizer.fontColor = goog.color.rgbArrayToHex(fillColorRgba);
     }
 
-    symbolizer.XOffset = textStyle.getOffsetX();
-    symbolizer.YOffset = textStyle.getOffsetY();
+    // Mapfish print allows offset only if labelAlign is defined.
+    if (goog.isDef(symbolizer.labelAlign)) {
+      symbolizer.labelXOffset = textStyle.getOffsetX();
+      symbolizer.labelYOffset = - textStyle.getOffsetY();
```

Comment:
Also, could you please remove the space character between the minus sign and `textStyle`?